website/www/site/content/en/documentation/basics.md

```diff
@@ -42,6 +42,14 @@ understand an important set of core concepts:
    them to a runner.
  * [_Runner_](#runner) - A runner runs a Beam pipeline using the capabilities of
    your chosen data processing engine.
+ * [_Window_](#window) - A `PCollection` can be subdivided into windows based on
```

Comment:
Technically not always "subdivided" per se. But TBH close enough, if you consider sliding windows to duplicate elements and then subdivide that result of that. I don't know how pedantic you want to be. I started writing this comment thinking we should change it but now I think keeping it as-is might be the best concise way of helping someone understand what windows are and the purpose they serve.